src/app/session/pages/register/register.component.html

```diff
@@ -1,23 +1,70 @@
-<form [formGroup]="registerForm">
-  <label>
-    <span>First Name</span>
-    <input type="text" formControlName="firstName" />
-  </label>
-  <label>
-    <span>Last Name</span>
-    <input type="text" formControlName="lastName" />
-  </label>
-  <label>
-    <span>Email</span>
-    <input type="email" formControlName="email" />
-  </label>
-  <label>
-    <span>Password</span>
-    <input type="password" formControlName="password" />
-  </label>
-  <label>
-    <span>Confirm Password</span>
-    <input type="password" formControlName="confirmPassword" />
-  </label>
-  <button type="submit" [disabled]="registerForm.invalid">Submit</button>
-</form>
+<div class="signUp">
+  <form [formGroup]="registerForm">
+    <div class="headerSignForm">
+      <div class='header-back'>
+        <a href="#" id="previousBtn">&#8249;</a>
+      </div>
+      <div class='header-title'>
+      <p id="signUpTitle">Sign Up</p>
+      </div>
+    </div>
+    <div class="bodySignForm">
+    <br />
+    <div style="display:flex">
+    <div>
+      <label for="firstName">First Name</label>
+      <mat-form-field>    
+        <input type="text" class="input-field" matInput placeholder="First Name" formControlName="firstName" required />
+        <mat-error *ngFor="let validation of registerForm_validation_messages.fistName">
+          <mat-error class="error-message" *ngIf="registerForm.get('firstName').hasError(validation.type) && (registerForm.get('firstName').dirty || registerForm.get('firstName').touched)">{{validation.message}}</mat-error>
```

Comment:
`*ngIf="registerForm.get('firstName').hasError(validation.type)"` is enough for this condition and the others